Components/CommentsList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByArticleId } from "../api";
+import { SingleCommentCard } from "./SingleCommentCard";
+import loadingGif from '../src/assets/loading.gif'
+
+export const CommentsList = () => {
+    const { article_id } = useParams();
```

Comment:
You should be able to pass this into the CommentsList as a prop from SingleArticleCard, rather than having to repeat it here 👍